examples/cli/pom.xml

```diff
@@ -16,8 +16,8 @@
   </description>
 
   <properties>
-    <maven.compiler.target>17</maven.compiler.target>
-    <maven.compiler.source>17</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
```

Comment:
Quarkus generally recommends using Java 17. Is there a reason to move this back to 11?

Reply:
Thank you for catching this!